src/lib/states.ts

```diff
@@ -126,6 +126,21 @@ export class InverterStates {
                 state: {id: 'dailyEnergyYield', name: 'Daily energy yield', type: 'number', unit: 'kWh', role: 'value.energy'},
                 register: {reg: 32114, type: ModbusDatatype.uint32, length: 2, gain: 100}
             },
+            {
+                interval: UpdateIntervalID.LOW,
+                state: {id: 'generatedenergytoday', name: 'Daily generated Energy', type: 'number', unit: 'kWh', role: 'value.energy.generated'},
```

Comment:
Pls change all role values accordingly to https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/dev/stateroles.md#values-numbers-read-only.

see comment: https://github.com/ioBroker/ioBroker.repositories/pull/3038#issuecomment-1843298637